src/main/java/org/wildfly/security/http/impl/FormAuthenticationMechanism.java

```diff
@@ -205,6 +206,16 @@ private void attemptAuthentication(HttpServerRequest request) throws HttpAuthent
     }
 
     private boolean authorize(String username, HttpServerRequest request) throws HttpAuthenticationException {
+        if (log.isTraceEnabled()) {
```

Comment:
Understood. My train of thought is to provide referential information between the login action and its associated session.